src/components/ChangePlan/reducers/changePlanReducer.js

```diff
@@ -0,0 +1,24 @@
+export const CHANGE_PLAN_ACTIONS = {
+  SET_LOADING: 'SET_LOADING',
```

Comment:
How about toggle instead of set? You don't set a value here, but toggle the one you have.
```suggestion
  TOGGLE_LOADING: 'TOGGLE_LOADING',
```